packages/mobile/src/harmony-native/components/Button/Button/Button.tsx

```diff
@@ -222,20 +223,21 @@ export const Button = (props: ButtonProps) => {
   }
 
   const animatedButtonStyles = useAnimatedStyle(() => {
-    if (isDisabled) return {}
     return {
       borderColor: interpolateColor(
         pressed.value,
         [0, 1],
         [dynamicStyles.default.border, dynamicStyles.press.border]
       ),
```

Comment:
do we want the border to be updating if the button is disabled?

Reply:
good call, that's not needed